src/snowflake/cli/app/telemetry.py

```diff
@@ -74,9 +76,18 @@ def _find_command_info() -> TelemetryDict:
         CLITelemetryField.COMMAND_OUTPUT_TYPE: ctx.params.get(
             "format", OutputFormat.TABLE
         ).value,
+        CLITelemetryField.PROJECT_DEFINITION_VERSION: _get_definition_version(),
     }
 
 
+def _get_definition_version() -> str:
+    from snowflake.cli.api.cli_global_context import cli_context
+
+    if cli_context.project_definition:
+        return cli_context.project_definition.definition_version
+    return "no_definition"
```

Comment:
Should it be `None` instead?

Reply:
I decided to use a more meaningful string, then when you build a dashboard you can see table `1 | 1.1 | 2.0 | no_definition`. Thought this is nicer. WDYT?